### ISSUE TITLE
improve incoming message synchronisation

### ISSUE THREAD

**Issue by realies** [EionRobb/purple-instagram#5]
it seems that incoming messages are delayed for significantly longer than when received via the original client and sometimes could take a while to resync missing messages when used via bitlbee

**Comment by EionRobb** [comment]
Yup, that's because we're polling for new messages rather than using the real-time MQTT stream

**Comment by realies** [comment]
is an MQTT implementation possible?

**Comment by EionRobb** [comment]
Given enough time, anything is possible 😛 

What might work better is using a 3rd party library that does instagrams MQTT and hooking into that instead of writing something from scratch like this plugin does
